store/buda/saga.js

```diff
@@ -121,10 +123,94 @@ function* getBudaPaymentHistory() {
   yield put(budaActions.finish());
 }
 
+function* postWithdrawal(action) {
+  const JS_MILISEC = 1000;
+  const BTC_TO_SAT = 100000000; // 1 BTC = 100,000,000 SAT
+  const DATE_OPTIONS = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  };
+
+  yield put(budaActions.start());
+  try {
+    const {
+      data: {
+        data: { response },
+      },
+    } = yield call(api.bitSplitWithdrawalApi, { ...action.payload });
+    if (response.data) {
+      yield put(
+        budaActions.setLastWithdrawal({
+          amount: response.data.amount / BTC_TO_SAT,
+          processed_at: new Date(
+            response.data.processed_at * JS_MILISEC
+          ).toLocaleDateString('es-ES', DATE_OPTIONS),
+        })
+      );
+      yield put(budaActions.syncBudaRejected('Transacción en proceso'));
+      action.callback();
+    } else {
+      yield put(budaActions.syncBudaRejected(response.message));
+    }
+  } catch (err) {
+    yield put(budaActions.syncBudaRejected('Hubo un error en el retiro'));
+  }
+  yield put(budaActions.finish());
+}
+
+function* postDeposit(action) {
+  const JS_MILISEC = 1000;
+  const BTC_TO_SAT = 100000000; // 1 BTC = 100,000,000 SAT
+  const DATE_OPTIONS = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  };
+
+  yield put(budaActions.start());
+  try {
+    const {
+      data: {
+        data: { response },
+      },
+    } = yield call(api.bitSplitDepositApi, { ...action.payload });
+    if (response.data) {
+      yield put(
+        budaActions.setLastDeposit({
+          amount: response.data.amount / BTC_TO_SAT,
+          processed_at: new Date(
+            response.data.processed_at * JS_MILISEC
+          ).toLocaleDateString('es-ES', DATE_OPTIONS),
+          expires_at: new Date(
+            response.data.lightning_invoice.expires_at * JS_MILISEC
+          ).toLocaleDateString('es-ES', DATE_OPTIONS),
+          payreq: response.data.lightning_invoice.payreq,
+        })
+      );
```

Comment:
desestructurando queda mas ordenado 
```suggestion
    if (response.data) {
    const { data : {amount, processed_at, lightning_invoice: {expires_at, payreq} } } = response.data;
      yield put(
        budaActions.setLastDeposit({
          amount: amount / BTC_TO_SAT,
          processed_at: new Date(
            processed_at * JS_MILISEC
          ).toLocaleDateString('es-ES', DATE_OPTIONS),
          expires_at: new Date(
            expires_at * JS_MILISEC
          ).toLocaleDateString('es-ES', DATE_OPTIONS),
          payreq,
        })
      );
```